packages/searchbase/src/index.js

```diff
@@ -1272,6 +1280,10 @@ Searchbase.defaultQuery = (value, options) => {
       finalQuery = {
         simple_query_string: Searchbase.shouldQuery(value, fields, options)
       };
+    } else if (options.queryString) {
```

Comment:
@jyash97 Set the priority to query string over search operators, check the RS code.